src/routes/accommodation.route.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import authMiddleware from '../middlewares/auth.middleware';
+import AccommodationController from '../controllers/accommodation.controller';
+import bookAccommodationValidation from '../validations/book-accommodation.validation';
+
+const router = express.Router();
+
+router.post('/book-accommodation', authMiddleware.checkUserLoggedIn, bookAccommodationValidation, AccommodationController.handleAccommodation);
```

Comment:
Since you are booking accommodation, the route should be `POST /accommodations` instead.